x86编译6.1.80失败，大佬们看看

### 详细叙述

touch /workdir/openwrt/staging_dir/target-x86_64_musl/root-x86/stamp/.kernel_installed
ERROR: module '/workdir/openwrt/build_dir/target-x86_64_musl/linux-x86_64/linux-6.1.80/net/sched/sch_dsmark.ko' is missing.
make[3]: *** [modules/netsupport.mk:1037: /workdir/openwrt/bin/targets/x86/64/packages/kmod-sched_6.1.80-1_x86_64.ipk] Error 1
make[3]: Leaving directory '/workdir/openwrt/package/kernel/linux'
time: package/kernel/linux/compile#2.91#0.40#3.30
    ERROR: package/kernel/linux failed to build.
make[2]: *** [package/Makefile:116: package/kernel/linux/compile] Error 1
make[2]: Leaving directory '/workdir/openwrt'
make[1]: *** [package/Makefile:110: /workdir/openwrt/staging_dir/target-x86_64_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/workdir/openwrt'
make: *** [/workdir/openwrt/include/toplevel.mk:231: world] Error 2

### 重复 issue

- [X] 没有类似的 issue

### 具体型号

x86

### 详细日志

touch /workdir/openwrt/staging_dir/target-x86_64_musl/root-x86/stamp/.kernel_installed
ERROR: module '/workdir/openwrt/build_dir/target-x86_64_musl/linux-x86_64/linux-6.1.80/net/sched/sch_dsmark.ko' is missing.
make[3]: *** [modules/netsupport.mk:1037: /workdir/openwrt/bin/targets/x86/64/packages/kmod-sched_6.1.80-1_x86_64.ipk] Error 1
make[3]: Leaving directory '/workdir/openwrt/package/kernel/linux'
time: package/kernel/linux/compile#2.91#0.40#3.30
    ERROR: package/kernel/linux failed to build.
make[2]: *** [package/Makefile:116: package/kernel/linux/compile] Error 1
make[2]: Leaving directory '/workdir/openwrt'
make[1]: *** [package/Makefile:110: /workdir/openwrt/staging_dir/target-x86_64_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/workdir/openwrt'
make: *** [/workdir/openwrt/include/toplevel.mk:231: world] Error 2

--- a/package/kernel/linux/modules/netsupport.mk
+++ b/package/kernel/linux/modules/netsupport.mk
@@ -1019,8 +1019,7 @@ define KernelPackage/sched
 	CONFIG_NET_EMATCH_META \
 	CONFIG_NET_EMATCH_TEXT
   FILES:=$(SCHED_FILES_EXTRA)
-  FILES+=$(LINUX_DIR)/net/sched/sch_dsmark.ko@lt6.2
-  AUTOLOAD:=$(call AutoLoad,73, $(SCHED_MODULES_EXTRA) sch_dsmark@lt6.2)
+  AUTOLOAD:=$(call AutoLoad,73, $(SCHED_MODULES_EXTRA))
 endef
 
 define KernelPackage/sched/description

